### PR TITLE
Feature/add simplecov

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @pixeltrix, 

This is not solving an issue but more of a _nice to have_. 

Screen Shot 2019-03-31 at 9 42 45 PM

I added `SimpleCov` as a development dependency. It's useful to find what parts of the app are least covered (ie. libraries)

Please check it out and let me know if you think it's useful.

Best,
Ernesto

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@
 
 # Add back any keep files that might have been excluded by the above
 !.keep
+
+# SimpleCov
+coverage
```
Comment (pixeltrix) [inline .gitignore]: Small nitpick - can you change this to:

```
# Ignore coverage reports from SimpleCov
/coverage/*
```

and put it on line 9 before `# Ignore logfiles and tempfiles.` - the `!.keep` should be the last thing and file is sort of alphabetical

**Comment by pixeltrix** [comment]
Happy to merge as long as it's a opt-in and not on by default (which it appears to be 👍 )

File: .gitignore (hunk 1 above)
Comment (etagwerker) [reply .gitignore]: @pixeltrix Sure thing. No problem.

**Comment by etagwerker** [comment]
@pixeltrix Cool, just updated the changes based on your feedback re: `.gitignore`

**Comment by pixeltrix** [comment]
@etagwerker thanks! 👍

**Pull request merged**